tools/dm/benchmark.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
Another thing: since we already have a folder named "benchmark" in the root directory, please **move this file to docs/benchmark and rename this file to dm-benchmark.md**. Also have checked with @GregoryIan.

Reply:
done, I have moved this file to `benchmark/dm-v1-alpha.md` which follows the name regulation under this folder